lms/djangoapps/course_home_api/toggles.py

```diff
@@ -51,7 +51,10 @@ def course_home_mfe_outline_tab_is_active(course_key):
 
 
 def course_home_mfe_progress_tab_is_active(course_key):
+    # Avoiding a circular dependency
+    from .models import DisableProgressPageStackedConfig
     return (
         course_home_mfe_is_active(course_key) and
-        COURSE_HOME_MICROFRONTEND_PROGRESS_TAB.is_enabled(course_key)
+        COURSE_HOME_MICROFRONTEND_PROGRESS_TAB.is_enabled(course_key) and
+        not DisableProgressPageStackedConfig.current(course_key=course_key).disabled is True
```

Comment:
`not xxx is True` feels confusing - why not just `not xxx`?

Reply:
updated